pkg/apis/v1/suite_test.go

```diff
@@ -44,7 +45,7 @@ func TestAPIs(t *testing.T) {
 
 var _ = BeforeSuite(func() {
 	env = coretest.NewEnvironment(coretest.WithCRDs(apis.CRDs...), coretest.WithCRDs(v1alpha1.CRDs...))
-	awsEnv = test.NewEnvironment(ctx, env)
+	awsEnv = test.NewEnvironment(options.ToContext(ctx, &options.Options{ClusterName: "test-cluster"}), env)
```

Comment:
Just wondering, why are we adding the cluster name here?

Reply:
This is avoid failing the context doesn't have cluster name when the describe call need derive the pointer of cluster name.